modules/apps/commerce/commerce-shipping-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -0,0 +1,7 @@
+.commerce-shipping-method-portlet {
+	.card {
+		&.panel-last-card {
```

Comment:
why can't we use the :last-child selector

Reply:
that's because the .card is not a last child (it's in the same container with the .dialog-footer)
`<form>
	...
	<div class="card d-flex flex-column panel-last-card">
		...
	</div>
	<div class="button-holder dialog-footer">
		...
	</div>
< /form>`

Using last-of-type css selector also doesn't work because it selects the last child only if it matches a condition (doesn't select the last element with the condition).
And in css, as far as I know, it's not even possible to style an element "followed" by something but only "preceded"